solutions/instances/main.tf

```diff
@@ -118,6 +119,7 @@ module "observability_instance" {
   # Routes
   activity_tracker_routes = [
     {
+      count      = var.provision_activity_tracker_routes == true ? 1 : 0
```

Comment:
Since AT routes is dependent on the targets, we should use a single variable to allow consumer to opt out of the service.
For ex-
`count = var.provision_atracker == true ? 1 : 0`